slasher/rpc/server.go

```diff
@@ -37,6 +39,10 @@ func (ss *Server) IsSlashableAttestation(ctx context.Context, req *ethpb.Indexed
 	ctx, span := trace.StartSpan(ctx, "detection.IsSlashableAttestation")
 	defer span.End()
 
+	log.WithFields(logrus.Fields{
+		"slot":    req.Data.Slot,
+		"indices": req.AttestingIndices,
+	}).Debug("Server attestation detection")
```

Comment:
Can you change this log to be a bit more descriptive? Maybe like "Received attestation via RPC"?

Reply:
changed